src/lib389/lib389/_constants.py

```diff
@@ -166,6 +166,13 @@
 DB2INDEX = 'db2index'
 DBSCAN = 'dbscan'
 
+# Task warnings
+class TaskWarning:
```

Comment:
Isn't there an enum type in python ?

Reply:
There is, to make matters worse we already import it for this file.